.travis.yml

```diff
@@ -15,6 +15,7 @@ env:
     - WITH_SCIPY=true WITH_CODECOV=true
 before_install:
   - pip install --upgrade setuptools pip
+  - pip install git+https://github.com/marrink-lab/vermouth-martinize.git#vermouth
```

Comment:
I'd move this to the install section, right above `pip install --upgrade .`. And add a comment why this is needed